maps/~mapsystem/maps.dm

```diff
@@ -93,7 +94,7 @@ var/const/MAP_HAS_RANK = 2		//Rank system, also togglable
 	var/list/lobby_tracks = list()                  // The list of lobby tracks to pick() from. If left unset will randomly select among all available /music_track subtypes.
 	var/welcome_sound = 'sound/AI/welcome.ogg'		// Sound played on roundstart
 
-	var/default_law_type = /datum/ai_laws/nanotrasen  // The default lawset use by synth units, if not overriden by their laws var.
+	var/default_law_type = /datum/ai_laws/asimov  // The default lawset use by synth units, if not overriden by their laws var.
```

Comment:
jesus isn't there a generic 4 laws to use rather than this monstrosity?

Reply:
Per convo on Discord - personally I like Asimov. I feel like the generic Bay lawsets are all very boring and bland - Asimov allows for more inround stupidity.

Bearcat doesn't use it as the default anyway.